CODEOWNERS

```diff
@@ -3,8 +3,13 @@
 
 # It uses the same pattern rule for gitignore file
 # https://git-scm.com/docs/gitignore#_pattern_format
+#
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @google/trillian-team will be requested for
+# review when someone opens a pull request.
+*                                @google/trillian-team
```

Comment:
Just curious to know how the group alias is set up? Maybe add a link (if it's available) to show members?

Reply:
Github auto links @google/trillian-team 
https://github.com/orgs/google/teams/trillian-team

Reply:
nice one, thanks!